msteams-platform/sbs-inmeeting-document-signing.yml

```diff
@@ -89,22 +95,32 @@ items:
     * Add messaging endpoint to the dev tunnel that you created.
 
     [!INCLUDE [Azure app registration](includes/get-started/azure-app-registration.md)]
+    > [!div class="nextstepaction"]
+    > [I ran into an issue](https://github.com/MicrosoftDocs/msteams-docs/issues/new?template=Doc-Feedback.yaml&title=%5BI+ran+into+an+issue%5D+Add+App+registration&&author=%40surbhigupta&pageUrl=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fmicrosoftteams%2Fplatform%2Fsbs-inmeeting-document-signing%3Ftabs%3Ddev%26tutorial-step%3D3&contentSourceUrl=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fmsteams-docs%2Fblob%2Fmain%2Fmsteams-platform%2Fsbs-inmeeting-document-signing.yml&documentVersionIndependentId=3a692774-83a3-eccd-a5c2-7a5f43207399&metadata=*%2BID%253A%2Be473e1f3-69f5-bcfa-bcab-54b098b59c80%2B%250A*%2BService%253A%2B%2A%2Amsteams%2A%2A)
 
     **Create a tunnel**
 
     # [dev tunnel](#tab/dev)
 
     [!INCLUDE [Tunnel](includes/get-started/dev-tunnel.md)]
+    > [!div class="nextstepaction"]
+    > [I ran into an issue](https://github.com/MicrosoftDocs/msteams-docs/issues/new?template=Doc-Feedback.yaml&title=%5BI+ran+into+an+issue%5D+Create+a+tunnel+with+dev+tunnel&&author=%40surbhigupta&pageUrl=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fmicrosoftteams%2Fplatform%2Fsbs-inmeeting-document-signing%3Ftabs%3Ddev%26tutorial-step%3D3&contentSourceUrl=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fmsteams-docs%2Fblob%2Fmain%2Fmsteams-platform%2Fsbs-inmeeting-document-signing.yml&documentVersionIndependentId=3a692774-83a3-eccd-a5c2-7a5f43207399&metadata=*%2BID%253A%2Be473e1f3-69f5-bcfa-bcab-54b098b59c80%2B%250A*%2BService%253A%2B%2A%2Amsteams%2A%2A)
 
     # [ngrok](#tab/ngrok)
 
     [!INCLUDE [Tunnel](includes/get-started/ngrok-tunnel.md)]
+    > [!div class="nextstepaction"]
+    > [I ran into an issue](https://github.com/MicrosoftDocs/msteams-docs/issues/new?template=Doc-Feedback.yaml&title=%5BI+ran+into+an+issue%5D+Create+a+tunnel+with+ngrok&&author=%40surbhigupta&pageUrl=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fmicrosoftteams%2Fplatform%2Fsbs-inmeeting-document-signing%3Ftabs%3Dngrok%26tutorial-step%3D3&contentSourceUrl=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fmsteams-docs%2Fblob%2Fmain%2Fmsteams-platform%2Fsbs-inmeeting-document-signing.yml&documentVersionIndependentId=3a692774-83a3-eccd-a5c2-7a5f43207399&metadata=*%2BID%253A%2Be473e1f3-69f5-bcfa-bcab-54b098b59c80%2B%250A*%2BService%253A%2B%2A%2Amsteams%2A%2A)
```

Comment:
Let's make the title consistent in all SBS guide

Create a tunnel with ngrok => Create a tunnel using ngrok

Reply:
Updated as suggested